inc/templates/edit-form.tpl.php

```diff
@@ -134,3 +134,20 @@
 		<# } #>
 	</div>
 </script>
+
+<script type="text/html" id="tmpl-shortcode-ui-field-color">
+	<div class="field-block">
+		<label for="{{ data.attr }}">{{ data.label }}</label>
+		<input type="text" name="{{ data.attr }}" id="{{ data.attr }}" value="{{ data.value }}" placeholder="{{ data.placeholder }}" data-default-color="{{ data.value }}"/>
+		<# if ( typeof data.description == 'string' ) { #>
+			<p class="description">{{ data.description }}</p>
+		<# } #>
+	</div>
+	<script>
+		jQuery('[id="{{ data.attr }}"]').wpColorPicker({
+			change: function() {
+				jQuery(this).trigger('keyup');
+			}
+		});
+	</script>
+</script>
```

Comment:
How would you go about putting the template in another file? Probably need your intervention on how to go about splitting this one since as of the moment all the template files are in the same place